.circleci/config.yml

```diff
@@ -1591,6 +1591,30 @@ workflows:
             - attach_workspace:
                 at: /
 
+  GCP-Daily-Services-Query-Restart-Performance-6N-6C:
```

Comment:
name should be `GCP-Weekly-Services-Query-Restart-Performance-6N-6C`

Reply:
Yeah, fixed.